src/index.html

```diff
@@ -6,9 +6,57 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
+    <header class="header">
+
+      <a class="logo" href="index.html">
+        <img src="images/logo.png" alt="moyo.logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="ul">
+          <li class="li_items">
+            <a class="a_items blue-recktangle item-1  is-active" href="index.html">Apple</a>
+          </li>
+
+          <li class="li_items">
+            <a class="a_items" href="index.html">Samsung</a>
+          </li>
+
+          <li class="li_items">
+            <a class="a_items" href="index.html">Smartphones</a>
+          </li>
+
+          <li class="li_items">
+            <a class="a_items a_item4"
+               data-qa="hover"
+               href="index.html">Laptops & Computers
```

Comment:
- Keep your attributes correctly formatted
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/2d28d935-2a6c-487c-8420-e7189856491a)
- Get rid of unnecessary class a_item4